### DISCUSSION TITLE
give bonus badges for Heroic TBC RDF

### DISCUSSION THREAD

**Discussion by trclocke** [chromiecraft/chromiecraft#5730] [Suggestions]
It hugely incentivizes queuing for anything instead of just specific dungeons, helping fill out groups. Won't matter as much for a minute with heroics being new and popular, but definitely something to think about. 8.8 gold and nothing else is pretty lame at the moment. 

edit: If not badges, at least SOMEthing useful. The current RDF reward at 70 is as good as worthless.

**Comment by balleny** [comment]
there is a daily, similar to how it is also for wotlk, giving additional (current content, not previous content) marks once a day

**Comment by Aethereus-ManaOnion** [comment]
I agree it would incentivize people to queue to RHC instead of specific dungeons

**Comment by andovcr** [comment]
From a user on Discord without a Github account:

passingby — Today at 12:13 PM
TXT @Andov : currently implementation of RDF doesn't emphasize the harder work done by the tank/heal leading to the longer queue time or if it ever pops.  Some even stop queueing for it all together leading to the feature useless at  heroics level.  Suggestion: provide a little more incentive to the tank/heal in a form of either 1 or 2 more badge or a random loot bag with a chance for blue item/gem/enchant/pet/mount or whatever it is.. they deserve it.  The community deserve a revamp RDF as it is dead.  You can test my theory out by queueing for heroics and see it for yourself how long it takes for your instance to pop if it ever does.  do it 3x and you will see that it is almost impossible to get one going before 30-1hr mark.

**Comment by andovcr** [comment]
In the interest of preserving the discussion within Github, here are responses from Nyeriah, Bench, & egovir:
<img width="650" alt="image" src="https://github.com/chromiecraft/chromiecraft/assets/58115742/3af600eb-0296-4536-92b2-3ac16a4385ee">

**Reply by Apathyxia** [reply]
Bench is on point. Personally, as a tank, I sit at almost 400 badges on my main character and I have no need to really even run any heroics on alts. All these badges are after getting all the gear I needed (including transmogs) and just helping friends getting queues. The few times I have queued on my own, I can sit for an hour in a queue without healers. This just incentivises me to wait for guildies to log in and queue with them. 

I can't think of an incentive for me to queue more as a tank. It is depressing to queue as a tank when DPS are in full green gears and you do more damage than them. This is mainly the aftermath of reducing the keys' requirement to honoured reputation. Frankly, I don't think having the heroic key qualifies you to the role you queue for, further emphasising Nyeriah's point.

**Comment by andovcr** [comment]
Personally, I don't think fear of players signing up for roles they're not suited for should prevent staff from developing solutions.  In all other loot disputes, staff depends on the user-base to self-police by using the /ignore list.  That seems both consistent and simple to apply here.  However, I do understand their unwillingness to devote resources towards a custom solution, when it is that same userbase that is creating the problem.

I think adding any unique rewards like pets, titles, costumes, mounts, etc. would be fun until the tanks/healers have accumulated the loot they are looking for, at which point they will again stop queuing (much like once they have bought everything they need with badges).

That said, I think it's a problem worth considering, since an active RDF is a major selling point for players looking to join the server.  I'm hopeful that people will keep contributing to the suggestion discussion, including staff (with their rationale for what has/hasn't worked in the past), so that one day it could be better.

**Comment by clovaek** [comment]
I'm a big fan of the mythic score system as a means of bad luck protection.
It allows me to get a missing item for my main spec, or big off spec items.

Awarding score when queuing and completing a random, not specific, dungeon would make more sense than awarding badges, which you really don't need that many of.

**Reply by Apathyxia** [reply]
I agree, this would be a good approach to it. There was some talk about having dungeon mythics for the daily heroic, but that never came to fruition. I think would be a nice compromise for it, and would definitely help queues as people would be able to get their items quicker. But, this should then also include blue items to be bought, not just epic ones.

**Comment by Superbolide1** [comment]
How about each RDF HC giving one mythic point per completion? The price of gear from mythic points can be increased, and the amount of mythic points from gruul/mag can be increased as well
